backend/src/apiserver/server/experiment_server.go

```diff
@@ -44,7 +44,12 @@ func (s *ExperimentServer) ListExperiment(ctx context.Context, request *api.List
 	if request.PageToken != "" {
 		opts, err = list.NewOptionsFromToken(request.PageToken, int(request.PageSize))
 	} else {
-		opts, err = list.NewOptions(&model.Experiment{}, int(request.PageSize), request.SortBy, request.Filter)
+		f, err := parseAPIFilter(request.Filter)
```

Comment:
not quite understand the if else statement here. does it means only one of pagetoken and filter can exist?
Are filtered results paginated?

Reply:
pagetoken encapsulates both sorting and filtering, so if you already have a non-empty page token, we ignore further filtering + sorting criteria.